salt/modules/keystone.py

```diff
@@ -175,7 +175,7 @@ def auth(profile=None, **connection_args):
         salt '*' keystone.auth
     """
     __utils__["versions.warn_until"](
-        "3001",
+        "Sodium",
```

Comment:
```suggestion
        "Magnesium",
```

Reply:
This one should *not* be removed yet - there's about a week's worth of work (at least) to write tests for they keystone state and have it use the keystoneng module instead. That will have to happen for the Magnesium release at this point.